modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -136,6 +141,21 @@ public int getAutosaveInterval() {
 		return _ddmFormWebConfiguration.autosaveInterval();
 	}
 
+	public Map<String, String> getAvailableExportFormats() {
```

Comment:
Hey @pedroqueiroz, I think this method should be part of the `DDMExporterFactory` class. What do you think?